BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Enums/BugsnagSpanCatagory.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace BugsnagUnityPerformance
+{
+    public enum BugsnagSpanCatagory
+    {
+        APP_START,
+        NAVIGATION,
```

Comment:
just wondering why we opted for this rather than `SCENE_LOAD`?

Reply:
No particular reason, happy to go with SCENE_LOAD if you think it's better?

Reply:
happy for others to weigh in on this one too, but all our Unity Performance docs etc refer to Scene Loads rather than Navigations

Reply:
Agreed - it's also scene load in the docs.

Reply:
decided to remove this enum and stick with span name comparison for now.